Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -1,3 +1,15 @@
+#region constants
+
+$script:discussionWebApiNS = "Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi"
+
+# Limit the number of issues to be posted to this number 
+$PostCommentsModule_MaxIssuesToPost = 100
```

Comment:
There is a construct for constants in Powershell but it is cumbersome so I'll just use a normal variable.